pulumi/rust_integration_tests/__main__.py

```diff
@@ -35,6 +35,15 @@ def _rust_integration_container_images(
     }
 
 
+def _rust_integration_container_versions(
+    container_images: Mapping[str, DockerImageId]
+) -> Mapping[str, DockerImageVersion]:
+    container_versions = dict()
+    for key, value in container_images.items():
+        container_versions[key] = value.split(":")[1]
+    return container_versions
+
+
```

Comment:
i feel like we could just centralize this on say docker_images.py

Reply:
Done